src/views/DrawerNavigatorItems.js

```diff
@@ -36,6 +37,7 @@ const DrawerNavigatorItems = ({
       const label = getLabel(scene);
       const accessibilityLabel = typeof label === 'string' ? label : undefined;
       const extraLabelStyle = focused ? activeLabelStyle : inactiveLabelStyle;
+      const extraItemStyle = focused ? activeItemStyle : itemStyle;
```

Comment:
I think we should introduce a `inactiveItemStyle` prop as well which will apply to inactive item, and apply `itemStyle` to both active and inactive item.

Also otherwise this will be a breaking change.